tests/files/features/.lagoon.yml.test

```diff
@@ -0,0 +1,53 @@
+docker-compose-yaml: docker-compose.yml
+
+environment_variables:
+  git_sha: 'true'
+
+environments:
+  branch/routes:
+    routes:
+    - node:
+      - customdomain-will-be-main-domain.com
+      - customdomain-will-be-not-be-main-domain.com
+  branch/cronjob:
+    cronjobs:
+    - name: echo /files/cron.txt multi
+      schedule: "* * * * *"
+      command: echo "$(date)" >> /files/cron.txt && printf '%s_%s' "CRONJOB" "MULTIPLE" >> /files/cron.txt
+      service: node
+    - name: echo /files/cron.txt single
+      schedule: "* * * * *"
+      command: printf '%s_%s' "CRONJOB" "SINGLE" >> /files/cron.txt
+      service: node
+    - name: test H/1
+      schedule: "H/1 * * * *"
+      command: printf '%s_%s' "CRONJOB" "H1" >> /files/cron.txt
+      service: node
+    - name: test */1
+      schedule: "*/1 * * * *"
+      command: printf '%s_%s' "CRONJOB" "STAR1" >> /files/cron.txt
+      service: node
+    - name: test native
+      schedule: "0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59 * * * *" # this will force Lagoon to create a native cronjob
+      command: printf '%s_%s' "CRONJOB" "NATIVE" >> /files/cron.txt
+      service: node
+    - name: test H
+      schedule: "H * * * *"
+      command: printf '%s_%s' "CRONJOB" "H" >> /files/cron.txt
+      service: node
+  branch/env-specific-template:
+    templates:
+      node: .lagoon.node-env-specific.yml
+
+ci-multiproject1:
```

Comment:
this also can be removed, as we don't use this at all for a deployment

Reply:
Done in 7e4bba47